charts/prometheus-elasticsearch-exporter/values.yaml

```diff
@@ -78,6 +78,7 @@ service:
 deployment:
   annotations: {}
   labels: {}
+  initContainers: []
```

Comment:
`initContainers` needs to move up in the root context, e.g. just before `service`.